cli/smartcontract/smart_contract.go

```diff
@@ -819,6 +856,144 @@ func contractDeploy(ctx *cli.Context) error {
 	return nil
 }
 
+// contractUpdate updates an existing smart contract.
+func contractUpdate(ctx *cli.Context) error {
+	var (
+		paramsStart   = 0
+		args          []string
+		nefBytes      []byte
+		manifestBytes []byte
+	)
+	args = ctx.Args().Slice()
+	if len(args) < 1 {
+		return cli.Exit(errNoScriptHash, 1)
+	}
+	contractHash, err := flags.ParseAddress(args[0])
+	if err != nil {
+		return cli.Exit(fmt.Errorf("invalid contract hash '%s': %w", args[0], err), 1)
+	}
+	paramsStart++
+
+	if ctx.String("in") == "" && ctx.String("manifest") == "" {
```

Comment:
Replace this check with `if nefBytes == nil && manifestBytes == nil`, move it under nef/manifest parsing. As a result, an access to `in` and `manifest` arguments will be performed once.